src/org/gh/Stud.java

```diff
@@ -0,0 +1,35 @@
+//  Think of package as a directory, we can import packages for reuse
+package org.gh;
+
+// extends Parent means we inherit from Parent
+// This gives Student the ability to use Parent methods and attributes
+// implements Printable means we must override the methods in that interface
+public class McdWork extends Person implements Printable{
```

Comment:
The class name (McdWork) must match the File Name (Stud.java).  Since they don't match it's an error